src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/DraftApprenticeship.cs

```diff
@@ -106,6 +106,21 @@ public void Merge(DraftApprenticeshipDetails source, Party modifyingParty)
                 FlexibleEmployment.EmploymentPrice = null;
                 FlexibleEmployment.EmploymentEndDate = null;
             }
+
+            ClearPriorLearningWhenStartDateBeforeAug2022();
+        }
+
+        private void ClearPriorLearningWhenStartDateBeforeAug2022()
+        {
+            if (StartDate < new DateTime(2022, 08, 01))
```

Comment:
Should this date be configurable?  (Mostly to make it easy to test)

We have used this configuration in PAS: https://github.com/SkillsFundingAgency/das-employer-config/pull/1295/files

Reply:
Yes, I agree.  The start date is user-entered information so we do not need to configure the cut-off as testing can enter different start date values for different scenarios.